modules/csv/csv_test.go

```diff
@@ -47,11 +57,11 @@ a,	b	c
 	e	f
 g	h	i
 j		l
-m	n,	
+m	n,\t
```

Comment:
looks unrelated

Reply:
It fixes the lint and unit test, otherwise some IDE would trim the last space to make unit test fail.

The file `csv_test.go` was changed in this PR (TrN interface) so I have to fix other related code.